src/appkit.cs

```diff
@@ -904,6 +904,22 @@ interface NSApplicationDelegate {
 		void UserDidAcceptCloudKitShare (NSApplication application, CKShareMetadata metadata);
 #endif
 	}
+
+	[Mac (10, 12, 1)]
+	[Category]
+	[BaseType (typeof(NSApplication))]
+	interface NSApplication_NSTouchBarCustomization
+	{
+		[Export ("isAutomaticCustomizeTouchBarMenuItemEnabled")]
+		bool GetAutomaticCustomizeTouchBarMenuItemEnabled ();
```

Comment:
Well I found at least one place NSResponder_NSTouchBarProvider where I inlined and things stopped working :(